app/code/Magento/InventoryCatalog/Plugin/InventoryConfiguration/IsSourceItemsAllowedForProductType/DisableVirtualTypePlugin.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryCatalog\Plugin\InventoryConfiguration\IsSourceItemsAllowedForProductType;
+
+use Magento\Catalog\Model\Product\Type;
+use Magento\InventoryConfiguration\Model\IsSourceItemsAllowedForProductTypeInterface;
+
+/**
+ * Disable Source items management for virtual product type
+ */
+class DisableVirtualTypePlugin
```

Comment:
Looks like after last discussion
We need to remove this plugin

Reply:
done